### ISSUE TITLE
Intrinsic Param Error with Olympus Camera

### ISSUE THREAD

**Issue by fallout2077** [alicevision/meshroom#310]
I have an Olympus Pen E-PM1 camera, and when I drag the photos into Meshroom I see that they're flagged with the "No Camera Intrinsic Parameters" error. I attached one of the images below so you folks can inspect the exif data yourself and see what's up. I also added "OLYMPUS IMAGING CORP.;E-PM1;17.3" into the sensor database, to no avail. 
![pb181117](https://user-images.githubusercontent.com/20364249/49710858-152ddd80-fc0a-11e8-9ed0-fbe9458a00b2.jpg)

**Comment by natowi** [comment]
The EXIF data might not match your sensor database entry.
Read [this](https://github.com/alicevision/meshroom/wiki/Add-Camera-to-database) for details.
If you keep getting this error, please attach the error log.

**Comment by fallout2077** [comment]
> The EXIF data might not match your sensor database entry.
> Read [this](https://github.com/alicevision/meshroom/wiki/Add-Camera-to-database) for details.
> If you keep getting this error, please attach the error log.

Thanks for the response, natowi. I just discovered that I can edit the EXIF data with Windows Explorer by simply selecting all the images and editing the properties in the sidebar. I copied and pasted the make and model as stated in the database into their proper entries and *voila* the images now work correctly with Meshroom! Fortunately I found this out, because a couple other posts led me to believe that only specialized software like EXIF Pilot would be able to edit the EXIF data, which isn't bad aside from the face that batch editing is a payware addon.
